google-cloud-scheduler/src/test/java/com/google/cloud/scheduler/v1beta1/it/ITSystemTest.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.scheduler.v1beta1.it;
+
+import static junit.framework.TestCase.assertEquals;
```

Comment:
Can you please use either `Assert.assertEquals` or `Truth.assertThat`. Because we follow that convention on the rest of the repositories.

I would still vote `Truth` higher than `Assert`

Reply:
I think truth is your favorite right? have you found or seen anything related to truth in `pom.xml`file.